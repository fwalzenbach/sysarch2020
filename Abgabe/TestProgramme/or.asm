addiu $1, $0, 73
addiu $2, $0, 42
or $3, $1, $2

addu $8 $0 $0
or $9 $8 $0

addiu $10 $0 737
addiu $11 $0 1311
or $12 $10 $11

