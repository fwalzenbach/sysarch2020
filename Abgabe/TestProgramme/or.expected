@1 0x00000049
@2 0x0000002a
@3 0x0000006b
@8 0x00000000
@9 0x00000000
@a 0x000002e1
@b 0x0000051f
@c 0x000007ff